content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm

```diff
@@ -548,6 +549,7 @@ button.btn.btn-default.multiselect-clear-filter{
 											/>
 											$tlang.getString("access.secured")
 										</label>
+										<a class="fa fa-info-circle" title="[$tlang.getString('access.secured.tooltip')] tabindex="0" data-placement="bottom" role="button" data-toggle="popover" data-trigger="focus" data-html="true" data-content="$tlang.getString('access.secured.tooltip')"></a>
```

Comment:
the quotes on the title attribute look off

Reply:
This was addressed, thanks @ottenhoff